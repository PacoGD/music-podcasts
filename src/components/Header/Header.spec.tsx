import { render } from "@testing-library/react";
import { Header } from ".";

jest.mock("../../hooks/store", () => ({
  useAppSelector: jest.fn(() => {
    ("");
  }),
}));

jest.mock("react-router-dom", () => ({
  Link: ({ to, children }: { to: string; children: React.ReactNode }) => (
    <a href={to}>{children}</a>
  ),
}));

describe("Header component", () => {
  test("snapshot", () => {
    const { container } = render(<Header />);
    expect(container).toMatchSnapshot();
  });
});
