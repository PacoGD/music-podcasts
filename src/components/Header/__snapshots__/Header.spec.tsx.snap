// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component snapshot 1`] = `
<div>
  <header>
    <div>
      <a
        href="/"
      >
        Podcaster
      </a>
    </div>
    <div />
  </header>
</div>
`;
